src/main/java/com/hms/flexyosisoftconnector/OSIsoftConnectorMain.java

```diff
@@ -48,10 +48,6 @@ public static void main(String[] args) throws IOException, JSONException {
     Logger.LOG_CRITICAL(
         OSIsoftConfig.CONNECTOR_NAME + " v" + OSIsoftConfig.CONNECTOR_VERSION + " starting");
 
-    // Start the webserver to accept json file
-    RestFileServer restServer = new RestFileServer();

```

Comment:
Should https://github.com/hms-networks/flexy-osisoft-connector/blob/main/src/main/java/com/hms/flexyosisoftconnector/dataserver/RestFileServer.java be removed as well here?